lib/components/screens/likes_retweets/likes_retweets_screen.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:harpy/components/components.dart';
+import 'package:harpy/components/screens/likes_retweets/retweets/cubit/retweets_cubit.dart';
+import 'package:harpy/harpy_widgets/harpy_widgets.dart';
+import 'package:provider/provider.dart';
+
+class LikesRetweetsScreen extends StatelessWidget {
```

Comment:
```suggestion
class RetweetsScreen extends StatelessWidget {
```

Let's keep it specific to retweets since we can't support Likes yet.
Also, maybe `Retweeters` makes more sense, since we only show the user of the retweet.